mmaction/datasets/ava_dataset.py

```diff
@@ -316,10 +316,13 @@ def dump_results(self, results, out):
 
     def evaluate(self,
                  results,
-                 metrics=('proposal', 'bbox'),
+                 metrics=('mAP'),
                  metric_options=None,
                  logger=None):
         # need to create a temp result file
+        assert len(metrics) == 1 and metrics[0] == 'mAP', (
+            'For evaluation on '
+            'AVADataset, you need to use metrics "mAP"')
```

Comment:
add this:
`See https://github.com/open-mmlab/mmaction2/pull/567 for more info.`